docs/sources/datasources/aws-cloudwatch/aws-authentication.md

```diff
@@ -6,31 +6,33 @@ aliases = ["/docs/grafana/latest/datasources/cloudwatch"]
 weight = 205
 +++
 
-# AWS authentication
+## AWS authentication
 
-You can use one of the following authentication methods. All of these methods are enabled by default. You can disable them if necessary if you have server configuration access. For more information, refer to [allowed_auth_providers]({{< relref "../../administration/configuration.md#allowed_auth_providers" >}}) documentation.
+Requests from a Grafana plugin to AWS is made on behalf of an [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) or an [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html). The IAM user or IAM role needs to have the permission policies needed in order to perform a certain api action. The policies needed are specific to each data source, so please refer to the specific data source documentation for details.
```

Comment:
```suggestion
Requests from a Grafana plugin to AWS are made by an IAM role or an IAM user. The IAM user or IAM role must have the associated permission and associated policies to perform certain API actions. Since these policies are specific to each data source, refer to the data source documentation for details.
```

Reply:
Looks good, but I think we should use the words `on behalf of` since that's how they usually describe this in the AWS docs.